assignments/objects.js

```diff
@@ -49,7 +84,18 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name": "Susan",
+  "age": 70,
+  "child": {
+    "name": "George",
+    "age": 50,
+    "grandchild": {
+      "name": "George", 
+      "age": 30
+    }
+  }
+}
```

Comment:
it's possible to define methods inside the object itself, this would be a good wayto solve this Stretch problem